src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -225,13 +225,13 @@ limitations under the License.
         The command is generated in response to a received unicast CopyScene command
       </description>
       <arg name="Status" type="ENUM8"/>
-      <arg name="GroupIdFrom" type="group_Id"/>
-      <arg name="SceneIdFrom" type="INT8U"/>
+      <arg name="GroupIdentifierFrom" type="group_Id"/>
+      <arg name="SceneIdentifierFrom" type="INT8U"/>
     </command>
   </cluster>
 
   <bitmap name="SceneFeatures" type="BITMAP32">
-    <cluster code="0x0006" />
+    <cluster code="0x0005" />
```

Comment:
Nit: `Scene Names` below has a space in my copy of the spec.

Reply:
Yes, we have decided we don't want spaces in identifiers and I will be filing spec issues to remove those in the spec.